src/main/java/seedu/duke/Ui.java

```diff
@@ -0,0 +1,152 @@
+package seedu.duke;
+
+public class Ui {
+    private Parser parser;
+
+    /**
+     * A constructor that returns an instance of Ui.
+     *
+     * @param tasklist The TaskList instance for this instance of Duke.
+     */
+    public Ui(TaskList tasklist) {
+        this.parser = new Parser(tasklist);
+    }
+
+    /**
+     * Greets the user and calls the Parser to open a scanner and start
+     * taking in user input.
+     */
+    public void introduction() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke \n" + "What can I do for you?");
+
+        parser.nextCommand();
+    }
+
+    /**
+     * Informs the user what task has been deleted and how many tasks are left.
+     *
+     * @param deleted The deleted task.
+     * @param size The number of tasks remaining.
+     */
+    public static void deleteText(String deleted, int size) {
```

Comment:
Will it be more comprehensive if the deleted variable naming change to this? Similar to some other methods variable naming
```suggestion
    public static void deleteText(String deletedTasks, int size) {
```